packages/core/src/analytics/analytics-step-tracker.ts

```diff
@@ -421,15 +421,18 @@ export default class AnalyticsStepTracker implements StepTracker {
             }
 
             return {
-                product_id: item.productId,
+                product_id: String(item.productId),
                 sku: item.sku,
                 price: item.salePrice,
                 image_url: item.imageUrl,
                 name: item.name,
                 quantity: item.quantity,
-                brand: itemsData[item.productId] ? itemsData[item.productId].brand : '',
                 category: itemsData[item.productId] ? itemsData[item.productId].category : '',
-                variant: (itemAttributes || []).join(', '),
+                variant: (itemAttributes || []).join(', ') || 'single-product-option',
+                ...(itemsData[item.productId] &&
```

Comment:
Indentation?

Reply:
Not sure what do you mean. Eslint doesn't indicate any issues